src/main/java/uk/ac/ebi/atlas/experimentpage/link/LinkToPride.java

```diff
@@ -6,23 +6,28 @@
 
 import java.text.MessageFormat;
 import java.util.Collection;
+import java.util.Collections;
+import java.util.ArrayList;
 import java.util.function.Function;
-
-import static java.util.Collections.singleton;
+import java.util.stream.Collectors;
 
 @Component
 public class LinkToPride extends ExternallyAvailableContent.Supplier<BaselineExperiment> {
-    private static final Function<BaselineExperiment, String> formatLabel =
-            e -> MessageFormat.format("PRIDE Archive: project {0}", e.getSecondaryAccessions().stream().findAny().get());
-    private static final Function<BaselineExperiment, String> formatLink =
-            e -> MessageFormat.format("https://www.ebi.ac.uk/pride/archive/projects/{0}", e.getSecondaryAccessions().stream().findAny().get());
+    private static final Function<String, String> formatLabel =
+            secondaryAccession -> MessageFormat.format("PRIDE Archive: project {0}", secondaryAccession);
+    private static final Function<String, String> formatLink =
+            secondaryAccession -> MessageFormat.format("https://www.ebi.ac.uk/pride/archive/projects/{0}", secondaryAccession);
 
-    private static final Function<BaselineExperiment, ExternallyAvailableContent.Description> createIcon =
+    private static final Function<String, ExternallyAvailableContent.Description> createIcon =
             formatLabel.andThen(label -> ExternallyAvailableContent.Description.create("icon-pride", label));
 
     @Override
     public Collection<ExternallyAvailableContent> get(BaselineExperiment experiment) {
-        return singleton(new ExternallyAvailableContent(formatLink.apply(experiment), createIcon.apply(experiment)));
+        return  experiment.getSecondaryAccessions().isEmpty() ? Collections.emptyList() :
+                experiment.getSecondaryAccessions()
+                        .stream()
+                        .map(accession -> new ExternallyAvailableContent(formatLink.apply(accession), createIcon.apply(accession)))
+                        .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
Make this more readable for the later coming developer
```suggestion
        var secondaryAccessions = experiment.getSecondaryAccessions();

        if (!secondaryAccessions.isEmpty()) {
            return getExternallyAvailableContents(secondaryAccessions);
        } else {
            return emptyContent();
        }
```